docker-compose.yml

```diff
@@ -5,15 +5,19 @@ web:
     - '443/tcp'
   volumes:
     - './web:/var/www/html'
+    - './drupal.make:/var/www/html/drupal.make'
     - './docker/settings.php:/var/www/html/sites/default/settings.php:ro'
+    - './docker/settings.local.php:/var/www/html/sites/default/settings.local.php:ro'
     - './docker/drupal-cron.conf:/etc/cron.d/drupal-cron.conf:ro'
     - './docker/prepare-environment.sh:/etc/my_init.d/prepare-environment.sh'
     - './docker/php.ini:/etc/php/5.6/mods-available/ding.ini'
+    - "${SSH_AUTH_SOCK}:/tmp/ssh-agent"
```

Comment:
I think on Mac this will simply emit a warning from Docker about the variable not being set, and Drush Make will then fail. That's what happens on Linux in any case, if key forwarding is not enabled.

But I'd still like to do this The Right Way.